index.html

```diff
@@ -0,0 +1,443 @@
+            <span>see all the articles</span>
+        </div>
+    </article>
+
+    <article class="design-systems">
+        <div class="design-sytems-header">
+            <div class="design-system-header-parent">
+                <div class="design-sytems-header-title">
+                    <h3>Design Systems</h3>
+                </div>
+                <div class="design-sytems-header-text">
+                    Meet Design Systems, a recent Smashing book in which Alla Kholmatova explores how to set up an effective design system to create great digital products. With common traps, gotchas and lessons learned.
+                </div>
+                <div class="design-sytems-header-btn">
+                    <button>Table of Contents ↬</button>
+                </div>
+            </div>
+        </div>
+        <div class="design-sytems-articles">
+            <div class="design-parent">
+                <div class="design-sytems-articles-item">
+                    <h3>
+                        New Front-End Challenges To Master
+                    </h3>
+                    <div class="design-systems-articles-img"></div>
+                </div>
+                <div class="design-sytems-articles-item">
+                    <h3>
+                        New Front-End Challenges To Master
+                    </h3>
+                    <div class="design-systems-articles-img"></div>
+                </div>
+                <div class="design-sytems-articles-item">
+                    <h3>
+                        New Front-End Challenges To Master
+                    </h3>
+                    <div class="design-systems-articles-img"></div>
+                </div>
+            </div>
+        </div>
+        <div class="design-systems-articles-2">
+            <div class="container-members-content-2">
+                
+                <div class="design-sytems-articles-item">
+                    <h3>
+                        New Front-End Challenges To Master
+                    </h3>
+                    <div class="design-systems-articles-img"></div>
+                </div>
+                
+                <div class="design-systems-articles-item-2">
+                    <div class="design-cm-text">
+                        <h3>1,651 smashing members</h3>
+                        <p> A friendly community that keeps this website alive. With books, webinars, discounts and networking — for for just 1 coffee a month. Join the family.</p>
+                        
+                    </div>
+                    <div class="design-member-circle"></div>
+                
+                    <button class="cta-btn design-btn">Explore features →</button>
+                </div>
+            </div>
+
+
+        </div>
+        <div class="see-all-latest-articles-title design-see-all">
+            <span>see all the articles</span> 
+        </div>
+    </article>
+
+    <footer class="footer">
```

Comment:
- [ ] The footer should contain centered **links**. Remember to add the `<a>` tags.